include/Debug.h

```diff
@@ -3,7 +3,7 @@
 #define DEBUG true
 #define UTIL_VARIABLE_FUNC_DEBUG false
 #define BUILD_PROGRAM_LINES_BRANCH_INFO false
-#define RUN_UTIL_FUNCTION_TESTS true
+#define RUN_UTIL_FUNCTION_TESTS false
```

Comment:
Why is this disabled here?

Reply:
Yeah it's just for debugging purposes. I'll re-enable the test